nodes/Comprehension.py

```diff
@@ -1 +1,16 @@
+"""
+Comprehension astroid node
+
+Attributes:
+    - target  (class)  typically a name or tuple node; the reference to use for
```

Comment:
This should probably be updated so the type is some kind of node, rather than "class."